server/models/Stakeholder.js

```diff
@@ -1,513 +1,258 @@
-    { filter: { where, order, limit, offset }, ...idFilters } = undefined,
-    url,
-  ) => {
-    let filter = {};
-    filter = FilterCriteria({ ...idFilters, ...where });
-
-    const options = {
-      ...QueryOptions({ limit, offset, ...order }),
-    };
+        const children = await this._stakeholderRepository.getChildren(
+          stakeholderCopy,
+        );
 
-    const { next, prev } = makeNextPrevUrls(url, filter, options);
+        stakeholderCopy.children = children.map((child) => {
+          const childCopy = { ...child };
+          childCopy.parents = [{ ...stakeholderCopy }];
+          childCopy.children = [];
+          return childCopy;
+        });
 
-    // get organization from old entity table, enter org id to insert it and it's children
-    // await getUUID(repo, 1);
+        return stakeholderCopy;
+      }),
+    );
+  }
 
+  async getAllStakeholders(filter, limitOptions) {
     let dbStakeholders;
     let count;
 
     if (Object.keys(filter).length > 0) {
-      const { stakeholders, count: dbCount } = await repo.getFilter(
-        filter,
-        options,
-      );
+      const { stakeholders, count: dbCount } =
+        await this._stakeholderRepository.getFilter(filter, limitOptions);
+
       dbStakeholders = stakeholders;
       count = dbCount;
     } else {
-      const { stakeholders, count: dbCount } = await repo.getAllStakeholders(
-        options,
-      );
+      const { stakeholders, count: dbCount } =
+        await this._stakeholderRepository.getAllStakeholders(limitOptions);
       dbStakeholders = stakeholders;
       count = dbCount;
     }
 
-    const stakeholders = await getRelationTrees(dbStakeholders, repo);
+    const stakeholders = await this.getRelationTrees(dbStakeholders);
 
     return {
       stakeholders:
         stakeholders &&
         stakeholders.map((row) => {
-          return StakeholderTree({ ...row });
+          return this.stakeholderTree({ ...row });
         }),
       totalCount: count,
-      links: {
-        prev,
-        next,
-      },
-    };
-  };
-
-const getAllStakeholdersById =
-  (repo, acctStakeholder_id) =>
-  async ({ filter: { where, order }, ...idFilters } = undefined, url) => {
-    let filter = {};
-    filter = FilterCriteria({ ...idFilters, ...where });
-
-    const options = {
-      ...QueryOptions({ limit: 100, offset: 0, ...order }),
     };
+  }
 
-    // create next and prev urls
-    const { next, prev } = makeNextPrevUrls(
-      `${url}/${acctStakeholder_id}`,
-      filter,
-      options,
-    );
-
-    const orgId = Number(acctStakeholder_id);
-    // get organization from old entity table
-    const id =
-      acctStakeholder_id === null || Number.isNaN(orgId)
-        ? acctStakeholder_id
-        : await getUUID(repo, orgId);
+  async getAllStakeholdersById(org_id, filter, limitOptions) {
+    const id = await this.getUUID(org_id);
 
     let dbStakeholders;
     let count = 0;
 
     if (Object.keys(filter).length > 0) {
```

Comment:
Same as above...
The filter is no longer required when I update the client to match this api so let's check for the filter before checking it's length  `filter && Object.keys(filter).length > 0`